### PR TITLE
Updating the default UI result

### PR DESCRIPTION
**Pull request by SparshithNR**
- Display a summary of the audit
- Provide an option to see the detailed view to see version violating the policy.
- Sample:
>Unsupported

Screen Shot 2021-02-16 at 11 08 08 AM

> Unsupported with --verbose
 Screen Shot 2021-02-16 at 11 07 28 AM

> Supported 
Screen Shot 2021-02-16 at 11 08 44 AM

### CODE REVIEW

File: lib/help.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ module.exports = chalk`
   {bold Options}
     {cyan --help, -h} this output
     {cyan --json, -h} output as json
+    {cyan --all} outputs detailed report of support audit
```
Comment (suchitadoshi1987) [inline lib/help.js]: I would recommend this to be 
`--verbose/-v`
OR
`--details/-d`

File: lib/lts/index.js [hunk 2]
```diff
@@ -101,13 +101,32 @@ function ltsVersions(_time, currentDate) {
     });
 }
 
+function getCurrentLts(ltsList) {
```
Comment (suchitadoshi1987) [inline lib/lts/index.js]: please add some documentation to explain what this function does

File: lib/lts/index.js [hunk 3]
```diff
@@ -101,13 +101,32 @@ function ltsVersions(_time, currentDate) {
...
+  let latestLts = '';
+  Object.keys(ltsList).forEach(version => {
+    const versionInfo = ltsList[version];
+    const today = new Date();
```
Comment (suchitadoshi1987) [inline lib/lts/index.js]: lets move this before `forEach`

File: lib/lts/index.js [hunk 4]
```diff
@@ -101,13 +101,32 @@ function ltsVersions(_time, currentDate) {
...
+    if (
+      new Date(versionInfo.start_date) <= today &&
+      today <= new Date(versionInfo.end_date) &&
+      new Date(versionInfo.maintenance_start_date) >= today
+    ) {
+      latestLts = versionInfo.versionRange;
+    }
+  });
+  if (latestLts) {
```
Comment (suchitadoshi1987) [inline lib/lts/index.js]: is the aim for this function to short circuit and return if the `latestLTS` is found?

**Review by suchitadoshi1987 — CHANGES_REQUESTED** [review]

File: lib/help.js (hunk 1 above)
Comment (SparshithNR) [reply lib/help.js]: `--verbose` makes sense and meets with a lot of conventions that already exist.

File: lib/lts/index.js (hunk 4 above)
Comment (SparshithNR) [reply lib/lts/index.js]: This function returns the latest LTS from the list we have in ember-cli-lts.json or node-lts.json. 
It also helps in throwing an exception when the above JSON file is not updated with the latest LTS.

File: lib/lts/index.js [hunk 5]
```diff
@@ -101,13 +101,43 @@ function ltsVersions(_time, currentDate) {
     });
 }
 
+/**
+ *
+ * @param {list of LTS version} ltsList
+ * Returns current LTS version from the list of all LTS available
```
Comment (suchitadoshi1987) [inline lib/lts/index.js]:
```suggestion
 * Returns current LTS version from the sorted list (old to new) of all available LTS versions.
```

File: lib/output/cli-output.js [hunk 6]
```diff
@@ -0,0 +1,256 @@
...
+        chalk` {dim ${name}}`,
+        chalk`{dim ${resolvedVersion}}`,
+        chalk`{dim ${latestVersion}}`,
+      ]);
+    }
+  });
+  return table.toString();
+}
+
+/* Keep the order of voilations summary output as
```
Comment (suchitadoshi1987) [inline lib/output/cli-output.js]:
```suggestion
/* Keep the order of violations summary output as
```

File: lib/output/messages.js [hunk 7]
```diff
@@ -0,0 +1,53 @@
+'use strict';
+const chalk = require('chalk');
+const { supportedRanges } = require('../time/index');
+
+const MILLSINQUARTER = 91 * 24 * 60 * 60 * 1000;
```
Comment (suchitadoshi1987) [inline lib/output/messages.js]: Please add a comment on the definition of this formula

File: lib/project/index.js [hunk 8]
```diff
@@ -113,7 +121,12 @@ module.exports = async function isInSupportWindow(projectRoot) {
               supportedRanges(info.time[info['dist-tags'].latest]),
             );
           }
-          supportChecks.push({ ...result, name, resolvedVersion });
+          supportChecks.push({
+            ...result,
+            name,
+            resolvedVersion,
+            latestVersion: info['dist-tags'].latest,
```
Comment (suchitadoshi1987) [inline lib/project/index.js:129]: will the property `dist-tags` always be present in the `info` object?

**Review by suchitadoshi1987 — CHANGES_REQUESTED** [review]

File: lib/output/cli-output.js (hunk 6 above)
Comment (SparshithNR) [reply lib/output/cli-output.js]: This change is not present in the latest

File: lib/lts/index.js (hunk 5 above)
Comment (SparshithNR) [reply lib/lts/index.js]: This summary is wrong. This function returns the latest LTS from the list of all LTS available.

File: lib/project/index.js (hunk 8 above)
Comment (SparshithNR) [reply lib/project/index.js:129]: Yeah, It must be present always.

**Pull request merged**